apps/api/src/app/inbox/usecases/update-preferences/update-preferences.usecase.ts

```diff
@@ -145,7 +145,7 @@ export class UpdatePreferences {
         channels: preference.channels,
         organizationId: command.organizationId,
         environmentId: command.environmentId,
-        subscriberId: command.subscriberId,
+        subscriberId: subscriber._id,
```

Comment:
**concern 👀**: We're not using this same `subscriber._id` in the `getSubscriberTemplatePreferenceUsecase` call. Why doesn't the command's `subscriberId` work?